lib/IAMClient.js

```diff
@@ -316,19 +316,19 @@ class VaultClient {
     /**
      * Update account custom attributes
      *
-     * @param {string} name - account name
+     * @param {string} accountName - account name
      * @param {object} customAttributes - custom attributes
      * @param {VaultClient~requestCallback} callback - callback
      * @returns {undefined}
      */
-    updateAccountAttributes(name, customAttributes, callback) {
-        assert(typeof name === 'string' && name !== '',
+    updateAccountAttributes(accountName, customAttributes, callback) {
+        assert(typeof accountName === 'string' && accountName !== '',
             'name parameter is required for account name');
         assert(typeof customAttributes === 'object');
         this.request('POST', '/', true, callback, {
             Action: 'UpdateAccountAttributes',
             Version: '2010-05-08',
-            name,
+            accountName,
```

Comment:
We need this as `name` due to OSIS.
If you want to add a type check, an alternate might be:
```suggestion
            name: accountName,
```

In Vault2 we do not support this API yet so tests will not run.